kasa/smart/smartdevice.py

```diff
@@ -252,6 +241,11 @@ def is_on(self) -> bool:
         """Return true if the device is on."""
         return bool(self._info.get("device_on"))
 
+    @property
+    def is_off(self) -> bool:
+        """Return true if the device is on."""
+        return not self.is_on
```

Comment:
It'd be better to move this implementation to the base class and leave it to the devices to implement only `is_on`.

Reply:
Done